decisions/log/0027-footprint-fragment.md

```diff
@@ -0,0 +1,112 @@
+    ```
+
+3. In the definition of `ProductFootprintFragment`, add
+    ```
+    It is RECOMMENDED to include at least the [=ProductIds=] in the PF Request Event request body.:
+    ```
+    followed by an example
+
+
+## Consequences
+
+1. As this ADR creates a non-breaking change, the Tech Specs Version number must be updated from 2.1.x to 2.2.y.
+
+## Appendix
+
+This appendix contains the text intended for the new section to be added to the Technical Specifications.
+
+### The General principles of Event processing
+
+1. A host system should strive to accept an event unless for the following reasons
+    1. event processing is not support by the host system
+    2. the syntax in the event is not supported (e.g. when the event failed “syntax checks” of the request host system)
+2. Whenever a host system’s `Event` endpoint returns an error response, no follow up response event (or error event) will be sent back to the original requester
+3. The requested host system will attempt to always send a response event to the requesting host system, **but the requesting host system should not rely on this behavior and retry after an appropriate amount of time by sending another new request event.**
+
+### Async Event Processing: Business Case 1 – Requesting Product Footprints
+
+#### Context and Assumptions
+
+- This business case is triggered by a data recipient if they wish to access a Product Footprint **they could not yet access through the `ListFootprints` API.**
+- The data recipient and data owner have a mutually agreed upon way to identify products
+
+#### Workflow
+
+1. The data recipient sends a `ProductFootprintFragment`  to the `Events Endpoint` of the data owner. The data recipient should always include 1 or more product ids in property `productIds`. The data recipient should limit each fragment to exactly 1 specific product (e.g. a specific type of apple instead of all apples that somebody is offering)
+    1. If the recipient is requesting a specific reporting period, it should include the reporting period accordingly (example …)
+    2. if the recipient is requesting a specific geography, it should include …
+    3. if the recipient has another need that is not covered above, the data recipient should <…>
+        1. Note: it is possible that this request will not be commonly understood by the data owner’s host system
+
+#### Cases
+
+- Case 0: The solution does not support this kind of processing
+    - either because the event processing does not exit at all, or the product footprint fragment as it is submitted by the data recipient is not supported by the requested host system
+    - The `events` endpoint responds with HTTP error code `400` and with a body with error code `NotImplemented`
+- Case 1: A PCF does not exist (yet) or a partially matching PCF exists
```

Comment:
Not sure what is meant with "partially matching PCF"? A PCF value matching two out of four attributes passed via the Fragment? Shouldn't we only return 100% matches?